articles/quickstart/native/react-native/00-login.md

```diff
@@ -195,3 +201,21 @@ Upon successful authentication the user's `credentials` will be returned, contai
 ::: note
 For more information on the `accessToken`, refer to [Access Token](/tokens/overview-access-tokens).
 :::
+
+### Log the User Out
+
+To log the user out, you remove the Access Token from state, and then log the user out from Auth0 authorization server.
```

Comment:
This is confusing. I'd reverse it, because if the step of calling clearSession fails, we should not remove the token from the state. What about...?

```suggestion
To log the user out, redirect them to the Auth0 log out endpoint by calling `clearSession`. This will remove their session from the authorization server. After this happens, remove the Access Token from the state. 
```